rocAL/rocAL/include/readers/image/file_source_reader.h

```diff
@@ -100,5 +102,7 @@ class FileSourceReader : public Reader {
     std::shared_ptr<MetaDataReader> _meta_data_reader = nullptr;
     void replicate_last_image_to_fill_last_shard();
     void replicate_last_batch_to_pad_partial_shard();
+    std::pair<RocalBatchPolicy, bool>  _last_batch_info;
```

Comment:
Change it to last_batch_policy_info

Reply:
Its not policy info. I think its better to keep _last_batch_info. This contains information as a pair of <last_batch_policy, last_batch_padded>